### PR TITLE
behave pager wrapper

### PR DESCRIPTION
**Pull request by meeuw**
## Description
behave pager wrapper

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: tests/features/steps/wrappers.py [hunk 1]
```diff
@@ -13,3 +13,8 @@ def expect_exact(context, expected, timeout):
         raise Exception('Expected:\n---\n{0!r}\n---\n\nActual:\n---\n{1!r}\n---'.format(
             expected,
             actual))
+
+
+def expect_pager(context, expected, timeout):
+    expect_exact(context, "{0}\r\n{1}{0}\r\n".format(
+        context.conf['pager_boundary'], expected), timeout=5)
```
Comment (j-bennet) [inline tests/features/steps/wrappers.py]: You're providing a `timeout` parameter, but not using it.

Comment (belartodmarinus) [inline tests/features/steps/wrappers.py]: Nice catch! I've fixed it, thanks.

Comment (meeuw) [inline tests/features/steps/wrappers.py]: Nice catch! I've fixed it, thanks.

**Comment by j-bennet** [comment]
👍

**Pull request merged**